odoo/addons/base/tests/test_mail_examples.py

```diff
@@ -510,7 +510,7 @@
 </div>
 """
 REMOVE_CLASS_IN = [
-    '<div style="color: #000000; font-size: 12pt; font-family: \'Times New Roman\'">',
+    '<div style="font-size: 12pt; font-family: \'Times New Roman\'; color: #000000">',
```

Comment:
That's actually a test fix, the original was following the iteration order of the dict created to normalise CSS properties (regardless of the source document), the style cleanup thing was converted to an ordered dict and the result check must now follow source ordering.